### PR TITLE
Adding test coverage to src/info/info_field.rs

### PR DESCRIPTION
**Pull request by alessandroasm**
As requested as an item on task #700

### CODE REVIEW

File: src/info/info_field.rs [hunk 1]
```diff
@@ -30,3 +30,50 @@ pub enum InfoType {
...
     License,
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    struct InfoFieldImpl {
+        pub val: String,
+    }
```
Comment (spenserblack) [inline src/info/info_field.rs:40]:
```suggestion
    struct InfoFieldImpl(&'static str);
```

File: src/info/info_field.rs [hunk 2]
```diff
@@ -30,3 +30,50 @@ pub enum InfoType {
...
+    impl InfoField for InfoFieldImpl {
+        const TYPE: InfoType = InfoType::Project;
+
+        fn value(&self) -> String {
+            self.val.clone()
+        }
```
Comment (spenserblack) [inline src/info/info_field.rs:46]:
```suggestion
        fn value(&self) -> String {
            self.0.into()
        }
```

File: src/info/info_field.rs [hunk 3]
```diff
@@ -30,3 +30,50 @@ pub enum InfoType {
...
+        fn title(&self) -> String {
+            String::from("title")
+        }
+    }
+
+    #[test]
+    fn test_info_field_get() {
+        let info = InfoFieldImpl {
+            val: String::from("test"),
+        };
```
Comment (spenserblack) [inline src/info/info_field.rs:57]:
```suggestion
        let info = InfoFieldImpl("test");
```

File: src/info/info_field.rs [hunk 4]
```diff
@@ -30,3 +30,50 @@ pub enum InfoType {
...
+
+        assert_eq!(info.get(&[]), Some(String::from("test")));
+    }
+
+    #[test]
+    fn test_info_field_get_none_when_type_disabled() {
+        let info = InfoFieldImpl {
+            val: String::from("test"),
+        };
```
Comment (spenserblack) [inline src/info/info_field.rs:66]:
```suggestion
        let info = InfoFieldImpl("test");
```

File: src/info/info_field.rs [hunk 5]
```diff
@@ -30,3 +30,50 @@ pub enum InfoType {
...
+
+        assert_eq!(info.get(&[InfoType::Project]), None);
+    }
+
+    #[test]
+    fn test_info_field_get_none_when_value_is_empty() {
+        let info = InfoFieldImpl {
+            val: String::from(""),
+        };
```
Comment (spenserblack) [inline src/info/info_field.rs:75]:
```suggestion
        let info = InfoFieldImpl("");
```

**Review by spenserblack — COMMENTED** [review]
This style is influence by my Go usage, but since this is a mock struct that exists to test a trait's provided method, I wonder if it makes sense to simplify the struct's definition like this.

**Comment by alessandroasm** [comment]
@spenserblack Awesome suggestion! I just pushed a commit with the changes. Thank you :+1:

**Pull request merged**